SmartPlaces.Facilities/README.md

```diff
@@ -1,27 +1,69 @@
 # Smart Facilities
 
-This folder contains the code, configuration, and documentation for the Smart Facilities Components. This includes several libraries that are created as Nuget Packages, and some sample code.
+This folder contains the code, configuration, and documentation for the SmartPlaces.Facilities Components. This includes several libraries that are also packaged as Nuget Packages, and some sample code.
 
-### Required Software
+## Required Software
 
-1. Visual Studio 2022+
-2. Visual Studio Code
+1. Visual Studio 2022+ or Visual Studio Code
 
-## Getting Started
+## Libraries
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+There are currently 4 library projects:
 
-1. Clone the repo
-3. Compile the Solution.
+| Name | Description | Read More |
+| --- | --- | --- |
+| OntologyManager | Defines base classes and helpers which allow mapping from one ontology to another | [Read More](./lib/OntologyMapper/README.md) |
+| OntologyManager.Mapped | Defines the mappings from the Mapped ontology to Rec and Willow ontologies | [Read More](./lib/OntologyMapper.Mapped/README.md) |
+| IngestioManager | Defines the process flow for copying one topology to another topology and using the Ontology Mapper to convert the source ontology to a target ontology | [Read More](./lib/IngestionManager/README.md) |
+| IngestionManager.Mapped | Refines specific functionality needed when using Mapped as an input topology | [Read More](./lib/IngestionManager.Mapped/README.md) |
+
+The nugets can be found on [Nuget.org](https://www.nuget.org/packages?q=Microsoft.SmartPlaces).
+
+### Getting Started with the Lib projects
+
+Each library folder has a stand-alone solution which includes a src and test project.
+
+Dependencies between the projects are taken via Nugets, which can be a little bit painful at times, but we are working to improve that experience.
+
+When updating the src project for any of the libs, please remember to:
+1. Update the Assembly Version and Package Version in the csproj as appropriate. The projects use [Semantic Versioning](https://semver.org/), so please follow the semantic versioning guidelines when updating.
+2. Add Unit tests for any new features you add or bugs you fix
+3. Ensure the documentation for the lib is up to date
+4. Upon completion of the pull request, sent a note to [smartplaces@microsoft.com](mailto:smartplaces@microsoft.com) to request that they publish the latest version of the Nuget packages. Response times should be under 2 business days.
```

Comment:
I'd recommend not exposing this to the world to avoid bots scraping the internet for emails. If we really want to expose an email, lets keep it to an alias dedicated to releases.